src/dmd/initsem.d

```diff
@@ -382,6 +382,7 @@ extern(C++) Initializer initializerSemantic(Initializer init, Scope* sc, Type t,
             return new ErrorInitializer();
         }
         uint olderrors = global.errors;
+        Expression currExp = i.exp;
```

Comment:
Either a comment or a more explanatory name is needed.

Reply:
A comment in source or an explaination here?

currExp stands for current Expression.

Yeah, I'm gonna add tests for this. I did so locally but I didn't know how to do it properly, so I'll try to read up on that.

Reply:
The aim of the comment is to show to future readers of the code *why* the change exists e.g. Does it fix a bug? Is it to work around an existing problem? This information gets lost over time sadly so the comments need to be good.

So a little explainer is needed

Reply:
I have updated the expected test output. Do I need to add more comments about why currExp was added? Should I reference the issue in bugzilla?

Reply:
thankyou, Yes and yes.

Reply:
Ok, fixing the comments, will push again soon

Reply:
Better?